k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_cluster.tpl

```diff
@@ -27,27 +27,34 @@ cluster:
         data: {{- toYaml $val.dataDevices | nindent 10 }}
     {{- end }}
     {{- end }}
+  {{- if .Values.cortxRgw.enabled }}
   - name: server_node
     components:
     - name: utils
     - name: hare
     - name: rgw
+  {{- end }}
+  {{- if .Values.cortxControl.enabled }}
   - name: control_node
     components:
     - name: utils
     - name: csm
       services:
       - agent
+  {{- end }}
+  {{- if .Values.cortxHa.enabled }}
   - name: ha_node
     components:
     - name: utils
     - name: ha
+  {{- end }}
   - name: client_node
     components:
-    - name: hare
+    - name: utils
     - name: motr
       services:
         - motr_client
+    - name: hare
```

Comment:
I updated this to match the latest cluster.yaml.sample file, in an attempt to try and use a client, but I don't think it had any effect.